### PR TITLE
added python syntax checking to Inspections section

### PR DESCRIPTION
**Pull request by tastyminerals**

### CODE REVIEW

File: site/source/doc/manual/editing.md [hunk 1]
```diff
@@ -179,6 +179,30 @@ Also shown in the above screenshot is the use of the `cursor-goto-inspection`
...
+    name = 'pep8',
+    factory = function() return 'pep8 -' end
+})
+
+howl.inspection.register({
+    name = 'pyflakes',
+    factory = function() return 'pyflakes' end
+})
+
+return howl.mode.configure('python', {
```
Comment (shalabhc) [inline site/source/doc/manual/editing.md]: I don't think the `return` is needed here?

File: site/source/doc/manual/editing.md [hunk 2]
```diff
@@ -179,6 +179,30 @@ Also shown in the above screenshot is the use of the `cursor-goto-inspection`
 command. This provides a list of all inspections available for the current
 buffer, and let's you easily navigate to them.
 
+### Code inspections for Python
+In order to enable Python syntax checking, you can use pep8 or pyflakes by
```
Comment (shalabhc) [inline site/source/doc/manual/editing.md]: Nit: needs additional newline after header.

File: site/source/doc/manual/editing.md [hunk 3]
```diff
@@ -179,6 +179,30 @@ Also shown in the above screenshot is the use of the `cursor-goto-inspection`
...
+    factory = function() return 'pyflakes' end
+})
+
+return howl.mode.configure('python', {
+    inspectors_on_idle = {'pep8', 'pyflakes'}
+})
+
+```
+
+![Buffer inspections](/images/screenshots/monokai/python-inspect.png)
```
Comment (shalabhc) [inline site/source/doc/manual/editing.md]: Call it 'Python buffer inspections'?

**Comment by nilnor** [comment]
I don't want to add this to the editing section, nor the user manual. The manual/editing section is for general information about the editor, whilst this is language specific configuration snippets. We could potentially still add this to the documentation page, but under a new header such as "How do I.." or something similar. Alternatively this could go into the Wiki. Thoughts?

**Comment by shalabhc** [comment]
We have a FAQ on the wiki where this can be added: https://github.com/howl-editor/howl/wiki/Howl-FAQ

**Pull request closed without merging**